flake.nix

```
@@ -1,12 +1,21 @@
 {
   description = "binplz";
 
-  inputs.nixpkgs.url = "github:NixOS/nixpkgs/release-22.05";
-  inputs.flake-utils.url = "github:numtide/flake-utils";
-  inputs.nixos-channel.url = "https://nixos.org/channels/nixos-22.05/nixexprs.tar.xz";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+    nixos-channel.url = "https://nixos.org/channels/nixos-22.05/nixexprs.tar.xz";
```

I got the following error when trying to get into the deploy shell:

```console
$ nix develop .#deploy
error: NAR hash mismatch in input 'https://nixos.org/channels/nixos-22.05/nixexprs.tar.xz?narHash=sha256-bpqNwyC5xrJz2H7oKABSoD+PISoULSnArKaCS+YP%2f8o=' (/nix/store/04x8d0hcm4zwjxsn5iq7cyfskv1778km-source), expected 'sha256-bpqNwyC5xrJz2H7oKABSoD+PISoULSnArKaCS+YP/8o=', got 'sha256-8VZOldZQrpCpGqtxIan9zG8Iz5Xrs+buYMWqdLYFYos='
```

It makes sense, since `nixexprs.tar.xz` will keep getting updated.

Adding a comment so it is easier to remember that you need to update the hash in the lockfile:

```suggestion
    # This URL isn't stable, so the first time you check out this repo,
    # all developers need to run `nix flake lock --update-input nixos-channel`.
    # See https://github.com/binplz/binplz.dev/issues/5
    nixos-channel.url = "https://nixos.org/channels/nixos-22.05/nixexprs.tar.xz";
```